packages/langchain/src/openai/chat.ts

```diff
@@ -0,0 +1,74 @@
+import { CallbackManagerForLLMRun } from '@langchain/core/callbacks/manager';
+import { BaseMessage } from '@langchain/core/messages';
+import type { ChatResult } from '@langchain/core/outputs';
+import { AzureChatOpenAI, AzureOpenAI } from '@langchain/openai';
+import { OpenAiChatClient as OpenAiChatClientBase } from '@sap-ai-sdk/foundation-models';
+import { mapLangchainToAiClient, mapResponseToChatResult } from './util.js';
+import type { OpenAiChatModelInput, OpenAiChatCallOptions } from './types.js';
+
+/**
+ * OpenAI Language Model Wrapper to generate texts.
+ */
+export class OpenAiChatClient extends AzureChatOpenAI {
```

Comment:
[q] I think Zhongpin changed the type names to include Azure as a prefix. We should probably do it for langchain package as well?